lib/generators/rpush_migration_generator.rb

```diff
@@ -44,6 +44,7 @@ def copy_migration
     add_rpush_migration('rpush_2_7_0_updates')
     add_rpush_migration('rpush_3_0_0_updates')
     add_rpush_migration('rpush_3_0_1_updates')
+    add_rpush_migration('add_pushy')
```

Comment:
The migration template has been changed, this should read `rpush_3_1_0_add_pushy`.